.github/workflows/build-and-test.yaml

```diff
@@ -65,3 +65,33 @@ jobs:
           fi
           cat $FILE
           curl -v -F json_file=@$FILE https://coveralls.io/api/v1/jobs
+
+  build-and-test-fedora:
+    name: fedora
+    runs-on: ubuntu-latest
+    
+    steps:
+      - name: Setup Podman
+        run: |
+          sudo apt update
+          sudo apt-get -y install podman
+          podman pull fedora:latest
+      - name: Get source
+        uses: actions/checkout@v3
+        with:
+          path: 'GOTCHA'
+      - name: Create container and run tests
+        run: |
+          {
+              echo 'FROM fedora:latest'
```

Comment:
Can you add a docker folder in .github folder and add this docker image there itself. I think it would be useful for other people trying gotcha using this recipe. 

So we can have a structure like .github/docker/fedora.Dockerfile.

Reply:
Done.